### ISSUE TITLE
add more tips like chatgpt website

### ISSUE THREAD

**Issue by Emt-lin** [logancyang/obsidian-copilot#686]
**Is your feature request related to a problem? Please describe.**
`add more tips like chatgpt website.`

![Obsidian 2024-09-30 16 30 27](https://github.com/user-attachments/assets/bfab170f-1c85-4827-b478-f2aaf00b3757)

**Describe the solution you'd like**
`example:`

![CleanShot 2024-09-30 at 16 33 16](https://github.com/user-attachments/assets/91999bc3-e78d-4b5b-8afc-bc6c0d64a186)

**Comment by logancyang** [comment]
This is indeed quite important for UX. I'm developing agentic RAG on another branch, once it's ready I'll add some use cases here that show the unique strength of agentic QA.

**Comment by logancyang** [comment]
Actually there's no need to put this on hold. The conversation starters can be implemented in parallel; the specific starters can differ depending on the situation.

**Comment by logancyang** [comment]
@Emt-lin This is a great idea and would you like to take it on yourself?

**Comment by zeroliu** [comment]
I'm thinking about what tips can be helpful here. Do you think it's overkill to have LLM analyze the current active doc and come up with tips based on the content?

**Comment by Emt-lin** [comment]
@zeroliu We can discuss it in the logancyang's discord channel.
